src/components/FileSelector/icons/error.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
+function ErrorIcon({ inheritFillColor }) {
+    const fillCircleColor = inheritFillColor ? 'currentColor' : '#FE4849';
```

Comment:
we don't need to do this check here instead we should
on
`circle fill="currentColor"`
and on
`path fill="#000" fillOpacity=".102"`